src/pages/[platform]/build-a-backend/more-features/analytics/analytics-migration-guide/index.mdx

```diff
@@ -0,0 +1,140 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Migrate from v5 to v6',
+  description: '',
+  platforms: [
```

Comment:
As a sanity check, is this platform list correct? I haven't used the new doc site too much, but should this only be platforms that have a v5 to v6 migration guide for Analytics?

Reply:
I kept the same list as the other pages under Analytics but I think what you are pointing out makes sense, we should have this for only the migration platforms. Updating the PR accordingly, thanks!

Reply:
Updated it to, 
```
[
    'javascript',
    'react-native',
    'angular',
    'nextjs',
    'react',
    'vue'
  ]
```